packages/slice-machine/lib/builders/utils/index.ts

```diff
@@ -0,0 +1,20 @@
+import { AnyWidget } from '@models/common/widgets/Widget'
+import { Media } from "@models/common/widgets/types";
+
+interface Widgets {
+  [x: string]: AnyWidget
+}
+
+export const findWidgetByConfigOrType = (widgets: Widgets, config: any, type: string) => {
+  const isALinkField = type === 'Link';
```

Comment:
I think you should do it like that 
```suggestion
  if (type === 'Link') {
    switch(config?.select) {
      case Media.document: return widgets.ContentRelationship
      case Media.media: return widgets.LinkToMedia
      default: return widgets.Link. (you forgot that one but it was the default case on the original)
     }
  }
  
  return widgets[type]
```

Reply:
Why not :) 

It make clearer the fact the special case is the Link

Reply:
I just thought of why you didn't added the default case, it would give the same result, my bad.
You don't need to do this if you don't want to.